### PR TITLE
Implemented blog sentiment analysis by sentence.

### PR DESCRIPTION
**Pull request by justness**
Sentiment analysis used to be split up by new line, and is now split by punctuation (. or ? or !). Another gitignore fix was put in, as direct edits to Github weren't pulled to my local working copy.
image

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/BlogSentimentServlet.java [hunk 1]
```diff
@@ -27,7 +28,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
                 newBlog = newBlog + s;
                 continue;
             }
-            newBlog = newBlog + "<br><br>" + s;
+            if(s.charAt(s.length()-1)=='>'){
```
Comment (mamessner) [inline portfolio/src/main/java/com/google/sps/servlets/BlogSentimentServlet.java:31]: A comment before this if statement explaining the situation you're catching would be helpful for other readers (I think this is basically looking for the end of a paragraph? not sure though)

File: portfolio/src/main/java/com/google/sps/servlets/BlogSentimentServlet.java [hunk 2]
```diff
@@ -39,27 +45,30 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         String line;
         ArrayList<String> processed = new ArrayList<String>();
 
+        String full = "";
         while ((line = reader.readLine()) != null) {
-            processed.add(line);
-        }
+            full = full + line;
+        } 
+        String[] sentences = full.split("(?<=\\.)|(?<=\\!)|(?<=\\?)|(?<=\\>)");
```
Comment (mamessner) [inline portfolio/src/main/java/com/google/sps/servlets/BlogSentimentServlet.java:52]: Nice! Regex is something I've always struggled with. I'm super impressed

**Comment by justness** [comment]
Made some small bugfixes re: cursor and fonts.

File: portfolio/src/main/java/com/google/sps/servlets/BlogSentimentServlet.java (hunk 1 above)
Comment (justness) [reply portfolio/src/main/java/com/google/sps/servlets/BlogSentimentServlet.java:31]: Yep, it's looking for things like </a> and </p>. Not sure if there's a better solution but visually this produces the right result for now.

**Comment by mamessner** [comment]
> Made some small bugfixes re: cursor and fonts.

looks good to me, merging

**Pull request merged**